### ISSUE TITLE
Escaping commas when saving on the forms

### ISSUE THREAD

**Issue by daleperforce** [thedevdojo/voyager#3374]
Simple bug, If you have a large discription that has lots of commas these are not escaped and means the add failes on new inserts of page or blog. Might want to conside escaping them and then saving the.

**Comment by fletch3555** [comment]
Version information is required for bug reports.  Please follow this template: https://github.com/the-control-group/voyager/blob/1.1/.github/ISSUE_TEMPLATE/bug_report.md

Also, it would be helpful if you provided any further information on how to reproduce this.  Can you provide example text that demonstrates the issue? What field did you enter it into? Perhaps a screenshot of the BREAD config for it?  Any error messages and/or stacktrace?

In short, this isn't a "simple bug" as we don't even know what field type you've submitted the value against.  There's nothing in our code that should be able to break, simply due to extra commas...

**Comment by emptynick** [comment]
Closing due to inactivity
